internal/wrapper/checkargs.go

```diff
@@ -0,0 +1,67 @@
+package wrapper
+
+import (
+	"errors"
+	"os"
+	"strings"
+
+	"github.com/Masterminds/semver/v3"
+)
+
+func checkStateCommand(args []string, version *semver.Version) error {
+	versionImport, _ := semver.NewConstraint(">= 1.5.0")
+	versionMoved, _ := semver.NewConstraint(">= 1.1.0")
+	versionRemoved, _ := semver.NewConstraint(">= 1.7.0")
+	STATE_COMMAND_VAR := "TF_DEMUX_ALLOW_STATE_COMMANDS"
+
+	if checkArgsExists(args, "import") >= 0 &&
+		versionImport.Check(version) {
+		if allowStateCommand(STATE_COMMAND_VAR) {
+			return nil
+		} else {
+			return errors.New("need set TF_DEMUX_ALLOW_STATE_COMMANDS=true for the 'import' command. Consider using Terraform configuration import block instead")
```

Comment:
```suggestion
			return errors.New("refusing to execute `import` command; consider using the Terraform `import` block instead, or set TF_DEMUX_ALLOW_STATE_COMMANDS=true.")
```

I'd also suggest making the above a constant and using `fmt.Sprintf` to substitute the command, instead of duplicating this.